### PR TITLE
converts about page CSS to use CSS Modules, deals with linting issues

### PR DESCRIPTION
**Pull request by advaithg**
quick change to add css modules to stylelint config
might have to update rules during `.module` conversion as well

edit: also converts `about` page to use `.module` scss

### CODE REVIEW

File: .stylelintrc.json
Comment (mattxwang) [inline .stylelintrc.json:2]: Any chance we can make this multi-line and get cleaner diffs? May be something we'll have to push down the line, and use a `.js` file so we can have trailing commas.

**Review by mattxwang — COMMENTED** [review]
Seems good to me; @advaithg, do you have a trivial test case with this config on new module-based code? Would be good to have confidence that we haven't broken CI yet.

**Comment by mattxwang** [comment]
Thanks for the quick change @advaithg, just wanted to check in on this:

> do you have a trivial test case with this config on new module-based code?

Does this work with the about page?

**Comment by advaithg** [comment]
> Does this work with the about page?

Will test it out by merging the about page branch into this one!

**Comment by mattxwang** [comment]
Just to check - now that we're not using the camelCase notation, do we still need `stylelint-config-css-modules`?

**Comment by advaithg** [comment]
> Just to check - now that we're not using the camelCase notation, do we still need `stylelint-config-css-modules`?

doesn't seem like it - lemme get rid of it

**Comment by mattxwang** [comment]
Also, not sure if this is just a GitHub quirk, but the file diff has a few changes that shouldn't be rooted in your commits - have you tried merging `main` into your branch, and/or doing some ~ rebasing ~?

**Comment by advaithg** [comment]
> Also, not sure if this is just a GitHub quirk, but the file diff has a few changes that shouldn't be rooted in your commits - have you tried merging `main` into your branch, and/or doing some ~ rebasing ~?

I think this is from merging the about branch into this branch, which are basically the results of merging main into this. I'll merge main once again just to be safe!

File: .stylelintrc.json
Comment (mattxwang) [inline .stylelintrc.json:21]: Let's include a comment as to why we made this change, for the future devs!

File: styles/App.scss
Comment (mattxwang) [inline styles/App.scss:28]: Let's remove this before merging!

**Review by mattxwang — COMMENTED** [review]
Left some small comments; also, is the fragmentation on the `about` page from an editor setting? Might be good to revert that and let editor soft-wrapping do its work instead.

**Comment by advaithg** [comment]
> is the fragmentation on the `about` page from an editor setting? Might be good to revert that and let editor soft-wrapping do its work instead.

I fragmented it because of the linter, which was giving me issues with lines being too long!

File: .stylelintrc.json
Comment (advaithg) [reply .stylelintrc.json:21]: I don't think json files support commenting

Comment (mattxwang) [reply .stylelintrc.json:21]: Ah great point, thought we had switched it to the `.js` file already. Let's table that 'till later, then.

**Review by mattxwang — APPROVED** [review]
Adjusted the PR title, but otherwise seems good to merge. I imagine we'll do another CSS passover later. Great work Advaith!

**Pull request merged**
